drivers/include/net/netdev.h

```diff
@@ -413,6 +414,47 @@ typedef struct netdev_driver {
                const void *value, size_t value_len);
 } netdev_driver_t;
 
+/**
+ * @brief   Convenience function for declaring get() as not supported in general
+ *
+ * @param[in] dev           ignored
+ * @param[in] opt           ignored
+ * @param[in] value         ignored
+ * @param[in] max_len       ignored
+ *
+ * @return  always returns `-ENOTSUP`
+ */
+static inline int netdev_get_notsup(netdev_t *dev, netopt_t opt,
+                                    void *value, size_t max_len)
```

Comment:
Is there any advantage of declaring this inline? It is always used as function pointer right?

Reply:
Not having to create a `netdev` module (and creating dependency relations for the requiring devices) where you would put the implementation ;-). Given that the number of devices that need it is not that large, it isn't worth the effort, but still reduces code duplication.

Reply:
There is no code compiled for the 'netdev' module (as its only a header), so my take was to simply declare it inline, taking the cost of some duplication for the rare case where there is more than one device type using these stubs compiled in.